mappings/net/minecraft/client/gui/screen/ingame/AnvilScreen.mapping

```diff
@@ -1,5 +1,5 @@
 CLASS net/minecraft/class_5672 net/minecraft/client/gui/screen/ingame/AnvilScreen
-	FIELD field_27756 BG_TEX Lnet/minecraft/class_4523;
+	FIELD field_27756 ANVIL_TEXTURE Lnet/minecraft/class_4523;
```

Comment:
[Upstream](https://github.com/FabricMC/yarn/blob/fd21c81c4ca0964220d9073018432f2ba642bfc9/mappings/net/minecraft/client/gui/screen/ingame/AnvilScreen.mapping#L3) uses only the term `TEXTURE` here. Also because of [this](https://github.com/Legacy-Fabric/yarn/blob/1.8.9/CONVENTIONS.md#conciseness) I think

Reply:
See `FurnaceScreen`. I decided to make it more or less consistent with this naming. Some screens have more than one texture. I'll admit that I did screw up the convention in some places.

https://github.com/Legacy-Fabric/yarn/blob/1.12.2/mappings/net/minecraft/client/gui/screen/ingame/FurnaceScreen.mapping#L2

Reply:
Reverted these changes.